src/EditorFeatures/Core/InlineHints/InlineHintsDataTaggerProvider.cs

```diff
@@ -92,6 +101,10 @@ protected override async Task ProduceTagsAsync(TaggerContext<InlineHintDataTag>
             var hints = await service.GetInlineHintsAsync(document, snapshotSpan.Span.ToTextSpan(), cancellationToken).ConfigureAwait(false);
             foreach (var hint in hints)
             {
+                // If we don't have any text to actually show the user, then don't make a tag.
```

Comment:
What scenario is this helping?

Reply:
when you have an error type.   i thought about filtering closer to the symbol.  But i prefer at this level because it means all empty tags will be stripped out.